public/locales/fr/status.json

```diff
@@ -70,7 +70,7 @@
   "no-record": {
     "cannot-give-status": {
       "description": "Nous ne pouvons pas vous donner l'état de votre demande pour le moment",
-      "because": "L’état de votre passeport n’est pas disponible dû au fait que :",
+      "because": "L’état de votre passeport n’est pas disponible pour les rasions suivantes :",
```

Comment:
Add an non-breaking space character `\u00a0` before colon in french instead of regular space. Revisit other places in French locale.